addons/sourcemod/scripting/l4d_tank_control_eq.sp

```diff
@@ -738,4 +924,104 @@ stock void LoadTranslation(const char[] translation)
 		SetFailState("Missing translation file %s.txt", translation);
 
 	LoadTranslations(translation);
-}
\ No newline at end of file
+}
+
+public int Native_GetWhosHadTank(Handle plugin, int numParams)
```

Comment:
Tag `public` can be stripped cuz it's not necessary. Same as other non-forward functions.